tools/tf2openapi/cmd/e2e_test.go

```diff
@@ -0,0 +1,190 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/onsi/gomega"
+
+	"github.com/kubeflow/kfserving/tools/tf2openapi/types"
+)
+
+func TestFlowers(t *testing.T) {
```

Comment:
Consider templatizing these tests.

Reply:
i.e., try and remove all the redundant logic for each test. You're basically mapping input to outputs. Check how this is done elsewhere in the KFServing

Reply:
done